tools/build_pytorch_libs.sh

```diff
@@ -298,16 +298,16 @@ function build_caffe2() {
 
 
   # Fix rpaths of shared libraries
-  if [[ $(uname) == 'Darwin' ]]; then
-      # root/torch/lib/tmp_install/lib
-      report "Updating all install_names in $INSTALL_DIR/lib"
-      pushd "$INSTALL_DIR/lib"
-      for lib in *.dylib; do
-          report "Updating install_name for $(pwd)/$lib"
-          install_name_tool -id @rpath/$lib $lib
-      done
-      popd
-  fi
+  #if [[ $(uname) == 'Darwin' ]]; then
```

Comment:
This was going to be removed by Anders in an earlier PR but I asked him to wait until I tested it here with the binaries. If all the Mac jobs pass then I will remove this